doc/dev/packaging.md

```diff
@@ -1,26 +1,25 @@
 # Azure packaging
 
-This article describes how to declare setup.py and all packaging information for packages inside the `azure` namespace
+This article describes the recommendation on how to define namespace packaging to release a package inside the `azure` namespace. Being inside the `azure` namespace meaning you have a service `myservice` that you want to import using:
```

Comment:
"...describes the recommendations for defining namespace packaging to..."

Reply:
"Being inside the azure namespace means that a service (e.g. myservice) can be imported using:"